reports_download.py

```diff
@@ -6,23 +6,24 @@
 from selenium.webdriver import DesiredCapabilities
 from selenium.webdriver.chrome.options import Options
 
-# User id.
-email_value = r'<Your reg number>'
+# User ID.
+email_value = r'<Your register number>'
 
 # Password for the account.
 pwd_value = r'<Password>'
 
 url = r'https://care.srmist.edu.in/srmktrada'
 
-# chrome default dowload path (it's better to create a new folder for this). 
-download_path = '<paht to download the reports to>'
+# Add Chrome default download path (it's better to create a new folder for this). 
+# Windows: C:/Users/<username>/Downloads/<folder_name>
+download_path = '<path to download the reports to>'
 
 num_que = int(input("Enter the index number of question till which to check for reports: "))
 
-# The options.add_argument is the defualt user profile that 
-# could be found in a path similar to:
+# The options.add_argument is the default user profile that 
+# can be found in a path similar to:
 # Linux distros: /home/<username>/.config/google-chrome/Default
-# Windows: C:\Users\<username>\AppData\Local\Google\Chrome\User Data\Default
+# Windows: C:/Users/<username>/AppData/Local/Google/Chrome/User Data/Default
```

Comment:
Are you sure windows uses forward slashes for it's paths ? Because i remember it using backward slashes.

Reply:
I just verified with my windows system, it's double backward slash and single forward slash, it worked when I checked with my chromedriver path.

Reply:
Awesome, that works then.